ui/components/src/TimeSeriesTooltip/TooltipContent.test.tsx

```diff
@@ -27,8 +27,8 @@ describe('TooltipContent', () => {
           seriesIdx: 0,
           datumIdx: 84,
           seriesName: 'Test node demo.do.prometheus.io:9100',
-          date: 'Dec 23, 2022, 1:53:00 PM',
-          x: 1671821580000,
+          dateMs: 1671821580000,
+          x: 1671821580,
```

Comment:
Wonder why the `x`  value changed for this one, but not the other tests

Reply:
I'm going to move this to a separate unit test to make this more explicit since this isn't directly related to this PR, sorry I should have commented about it! 

While in this code, I remembered [this edge case](https://github.com/perses/perses/blob/v0.29.1/ui/components/src/TimeSeriesTooltip/focused-series.ts#L63-L65)  in focused-series.ts. One place where we were consuming LineChart internally, didn't pass x values as milliseconds and I still wanted the tooltip to work so I added a condition that checked for this. There isn't a unit test though so wanted to confirm that functionality still worked, but I'll update this to be a new test case!

Reply:
I need to check if this is even an issue internally anymore, I think the results _are_ being converted to milliseconds now, so maybe [that conversion](https://github.com/perses/perses/blob/v0.29.1/ui/components/src/TimeSeriesTooltip/focused-series.ts#L64) isn't really needed 🤔